.github/workflows/update_nightly.yml

```diff
@@ -0,0 +1,19 @@
+on:
+  schedule:
+  - cron: '7 1 * * *'
+jobs:
+  make_nightly:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v4
+
+    - name: merge_to_nightly
+      uses: robotology/gh-action-nightly-merge@1.4.0
+      with:
+        stable_branch: 'develop'
+        development_branch: 'nightly'
```

Comment:
is this correct?

Reply:
according to their documentation, yes: https://github.com/robotology/gh-action-nightly-merge . I have opinions about their use of stable and development in this context, but *shrug*

Reply:
no. here it is
```
      with:
        stable_branch: 'master'
        development_branch: 'devel'
```

Reply:
Hmm am I parsing "Automatically merge the stable branch into the development one " wrong?

Reply:
aaah sorry my bad.